### DISCUSSION TITLE
Beautify Extension

### DISCUSSION THREAD

**Discussion by Abdulwahabcse** [phcode-dev/phoenix#1866] [Q&A] [answered]
I Installed the Beautify extension into the Phoenix code as I usually do while I was using the Brackets code editor but unfortunately, when I click it, it doesn't beautify my code herein Phoenix code but it still works with the brackets editor.  please what's the problem?

**Comment by abose** [comment, marked as answer]
Phoenix code already has integrated code beautify and no extension installation is needed. Please see https://docs.phcode.dev/docs/Features/beautify-code/

**Comment by Abdulwahabcse** [comment]
Thanks for your feedback. I've checked. It works
